rosbag2_to_video/api.py

```diff
@@ -30,25 +30,49 @@ def get_stamp_from_image_msg(image_msg):
 
 def add_arguments_to_parser(argparser):
     argparser.add_argument(
-        'bagfile', help='Path to the bag (currently only supports sensor_msgs/msg/Image types)')
-    argparser.add_argument('-t','--topic', required=True, help='Name of the image topic')
-    argparser.add_argument('-o','--output', required=True, help='Output filename (currently only supports MP4)')
+        'bagfile',
+        help='Path to the bag'
+    )
+    argparser.add_argument(
+        '-t',
+        '--topic',
+        required=True,
+        help='Name of the image topic (currently only supports sensor_msgs/msg/Image or sensor_msgs/msg/CompressedImage types)' # noqa E501
```

Comment:
Avoid using `noqa`, except when strictly necessary

```suggestion
        help=(
            'Name of the image topic (currently only supports sensor_msgs/msg/Image or '
            'sensor_msgs/msg/CompressedImage types)')
```